### PR TITLE
Refactioring examples header

### PR DESCRIPTION
**Pull request by gela-d**
Before:

After:

### CODE REVIEW

**Comment by gela-d** [comment]
@tadatuta @tavriaforever

File: src/blocks/common.blocks/block-example/__link/block-example__link.styl [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 
     position: relative;
 
     display: inline-block;
 
-    margin-right: 15px;
+    padding: 0 8px;
 
     color: #8f8f8f;
+    border-right: 1px solid #e5e5e5;
```
Comment (tavriaforever) [inline src/blocks/common.blocks/block-example/__link/block-example__link.styl]: I offer to add this color into https://github.com/bem/bem-site-engine/blob/dev/src/blocks/common.blocks/styling/styling.styl

File: src/blocks/common.blocks/block-example/__link/block-example__link.styl [hunk 2]
```diff
@@ -1,15 +1,16 @@
 .block-example__link
 {
     font-size: 14px;
-    line-height: 1;
+    line-height: 32px;
```
Comment (tavriaforever) [inline src/blocks/common.blocks/block-example/__link/block-example__link.styl]: I think, that much better use multiplier values for line-height `1; 1.5; 2; etc` that lock values on px

File: src/blocks/common.blocks/block-example/__link/_icon/block-example__link_icon_blank.styl [hunk 3]
```diff
@@ -1,5 +1,6 @@
...
-    background-size: 20px;
+    background-image: url(block-example__link_icon_blank.png);
+    background-position: 4px 5px;
```
Comment (tavriaforever) [inline src/blocks/common.blocks/block-example/__link/_icon/block-example__link_icon_blank.styl]: In best practices, background-position must be `top left` or `0 0` and make position by use css properties `left,top, etc`

**Comment by tavriaforever** [comment]
The interface was better!

**Comment by gela-d** [comment]
@tavriaforever done all for corrections.

> The interface was better! 

What exactly was better?

**Comment by tavriaforever** [comment]
> What exactly was better?

I meant - now better

**Pull request merged**